docs/ubuntu1604.md

```diff
@@ -6,7 +6,7 @@ Ubuntu 16.04 for Uno.
 
 ```bash
 #install stuff
-sudo apt install openjdk-8-jdk maven git openssh-server wget libxml2-utils make g++
+sudo apt install openjdk-11-jdk maven git openssh-server wget libxml2-utils make g++
```

Comment:
I don't think java 11 is available in the default Ubuntu 16.04 repos.  It is for 18.04.

Reply:
Yeah, after some google checks, it appears you are correct. My mistake. Do you think it is best if I create a separate doc for 18.04 or newer and keep this one as it was?

Reply:
> Do you think it is best if I create a separate doc for 18.04 or newer and keep this one as it was?

I am in favor of that.